docs/running-clusterfuzzlite/github_actions.md

```diff
@@ -308,15 +295,13 @@ jobs:
 ```
 {% endraw %}
 
-You can copy and paste the above file and pruning should just work.
-But here is where `storage-repo`, `storage-repo-branch`, and
-`storage-repo-branch-coverage` come in handy.
+Optionally, edit the following fields to view coverage reports at
```

Comment:
Hmmm... do we tell users they should change OWNER and STORAGE-REPO-NAME to the appropriate values for their storage repo?

Reply:
I've added a third line: 

-  set `storage-repo` ([instructions here])
- set `storage-repo-branch-coverage` to "gh-pages" (the
default)
- set `owner` and `storage-repo-name` to the appropriate values for your storage repo

Does the first line (set `storage-repo`) still make sense now? Maybe it should be "set up a storage repo" with the link to the #storage-repo section?

Reply:
What you have makes sense. But I wonder if it should be in a more general place. Don't worry I can fix this.